### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by DevKingofEarth** [super-productivity/super-productivity#2691] [Ideas]
A customisation feature for setting background would be cool, make the app more oled friendly.

**Comment by ghost** [comment]
It would be really nice : )

**Reply by johannesjo** [reply]
This is now possible. You can use background images from unsplash.

**Comment by johannesjo** [comment]
Thanks for the suggestion. If you are crafty you can already do this by using a custom stylesheet: https://github.com/johannesjo/super-productivity-themes

**Reply by DevKingofEarth** [reply]
Thank you Sir, I will try it, it is just I am normal folk, just started learning Android App development. I am student in India, so, I will make it once I am free, but appreciate your response and this beautiful work! My hats off!
